### DISCUSSION TITLE
Empty nodes for spacing and grouping

### DISCUSSION THREAD

**Discussion by euu2021** [freeplane/freeplane#1470] [Show and tell]
The #1465 discussion gave me the idea to share this little trick that I use.

---
I like to use empty nodes to create simple and quick grouping of nodes, in situations where the label of the grouping can be implicit. For example, in this situation I want to group the actual fruits to make it apart of the other information, but I don't need to bother using a label in the grouping:

![image](https://github.com/freeplane/freeplane/assets/77707706/1781c82a-e0c3-4c29-8ad7-163d882da0c5)

So I use a style to a node in order to make it look like a simple line extension (I will call it "just line" style). To me, it's visually more clean than using a usual node without any text. Comparison:

![image](https://github.com/freeplane/freeplane/assets/77707706/7c577bcb-884e-4ced-9a9e-3820b0d91e2e)

Now, it's possible to manually apply the "just line" to the empty nodes, but it's easier to create a conditional style, so every empty node gets automatically styled as "just line". 

The relevant settings of the "just line" style are those:

![image](https://github.com/freeplane/freeplane/assets/77707706/9b53076b-9041-4443-a7df-c58f6f1b8384)

And the conditional style is like that (the condition is simply Node Core is equal to nothing):

![image](https://github.com/freeplane/freeplane/assets/77707706/72ac0099-6665-4442-bb0d-697d84cbacec)

This can also be used to create space, in order to keep vertical parallelism between nodes:
![image](https://github.com/freeplane/freeplane/assets/77707706/aa3c19df-a72d-4f40-adc4-07f7b24d4e37)

Example map:
[EXAMPLE grouping.zip](https://github.com/freeplane/freeplane/files/12919964/EXAMPLE.grouping.zip)

**Comment by adamfreeplaneuser01** [comment]
Just tried this out. Worked perfectly!

**Comment by adamfreeplaneuser01** [comment]
The only thing I am wondering is if there would be a way to automatically have the "line nodes" appear when there are blank spots before a summary node?
Ex:
<img width="519" alt="dsafdsfs" src="https://github.com/freeplane/freeplane/assets/147225942/0bdfd543-8bc9-4083-ba4d-1406ab25d1c6">

**Reply by adamfreeplaneuser01** [reply]
Obviously, I could just add them in manually now, but figured it would be worth asking

**Reply by euu2021** [reply]
Maybe there is a way, but I don't know. I don't use summary nodes, so I don't know the advanced tricks of using it.

**Comment by Foxxsayy** [comment]
Is there just an option in Freeplane to "Align Each Level of Nodes with at the End of the Previous Level's longest Node" for everything visible on the Mind Map? Basically just aligning by levels - for example, the beginning of every third node on the map would be aligned with each other, the beginning of every 4th node would be aligned with each other, etc.?
